functions/sendMail.js

```diff
@@ -0,0 +1,49 @@
+"use strict";
+const nodemailer = require("nodemailer");
+
+exports.handler = function (event, context, callback) {
+  const { img, desc, river, rapid } = JSON.parse(event.body);
+
+  async function main() {
+    // create reusable transporter object using the default SMTP transport
+    let transporter = nodemailer.createTransport({
+      host: "smtp.zoho.com",
+      port: 465,
+      secure: true, // true for 465, false for other ports
+      auth: {
+        user: "wetexittest@zohomail.com",
+        pass: "pullout123",
```

Comment:
If this is a real email address, then this is not secure to push to a public github repo.. could maybe look into storing this as an environment variable.

Reply:
Good idea!